recipes/pymks/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "pymks" %}
+{% set version = "0.3.4a7" %}
+{% set sha256 = "030d56481debaa4e68feee4102b1189e5f999c8aa427f4c431fa94c766f35f80" %}
+{% set hash = "ab2f93f57eeaf23b6d06d282dca60438c9a959e2ff97d0ecbdcf69acdfdf3a43" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/{{ hash[:2] }}/{{ hash[2:4] }}/{{ hash[4:] }}/{{ name }}-{{ version }}.tar.gz
```

Comment:
The URL from the example seems to work for me. I tried:

`https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz`

which is translate to the URL below after the jinja is applied and I got the right file.

`https://pypi.io/packages/source/p/pymks/pymks-0.3.4a7.tar.gz`